gui/wxpython/location_wizard/wizard.py

```diff
@@ -171,14 +162,26 @@ def __init__(self, wizard, parent, grassdatabase):
         self.locTitle = ''
 
         # text controls
-        self.tgisdbase = self.MakeStaticText(grassdatabase, size=(-1, -1))
-        self.tlocation = self.MakeTextCtrl("newLocation", size=(300, -1))
+        self.tgisdbase = self.MakeTextCtrl(grassdatabase, size=(400, -1), style = wx.TE_READONLY | wx.TRANSPARENT_WINDOW)       
+        self.tlocation = self.MakeTextCtrl("newLocation", size=(400, -1))
         self.tlocation.SetFocus()
         self.tlocation.SetValidator(
             GenericValidator(
                 grass.legal_name,
                 self._nameValidationFailed))
         self.tlocTitle = self.MakeTextCtrl(size=(400, -1))
+        
+        # text for required options
+        self.required_txt = self.MakeLabel("*")
+        self.required_txt.SetForegroundColour(wx.RED)
+        self.required_txt.SetToolTip(_("This option is required"))
+        
+        # text for optional options
+        self.optional_txt = self.MakeLabel("(optional)")
+        italics = wx.Font(10, wx.DEFAULT, wx.ITALIC, wx.NORMAL)
+        self.optional_txt.SetFont(italics)
+        self.optional_txt.SetForegroundColour("gray")
```

Comment:
I would recommend here using system colors instead of 'gray': `wx.SystemSettings.GetColour(wx.SYS_COLOUR_GRAYTEXT)` because people have different window styles.